kernel/syscall.c

```diff
@@ -3858,6 +3882,21 @@ static long _syscall(void* args_)
             ret = (long)myst_mremap(
```

Comment:
need to update the process mappings to remove the old and add the new... even if it is a partial remap

Reply:
Again, not part of the half-measure solution. We will be removing the process-mappings as noted above once we rework ``/proc/self/maps``.